modules/4337/contracts/Safe4337Module.sol

```diff
@@ -258,16 +258,17 @@ contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandle
     function _validateSignatures(PackedUserOperation calldata userOp) internal view returns (uint256 validationData) {
         (bytes memory operationData, uint48 validAfter, uint48 validUntil, bytes calldata signatures) = _getSafeOp(userOp);
 
+        // checkSignatures function in Safe contract does not force a fixed size on signature length. A malicious actor can append additional bytes in the signature than needed and force gas usage to reach verificationGasLimit. _checkSignatureLength ensures that there are no additional bytes in the signatures than required.
```

Comment:
Nit: I think all other comments have been split at either 100 or 120 column mark, and we have some markdown-y ways for referencing identifiers (so `checkSignatures`). Can you keep this long comment format consistent here as well?

The reason for breaking comments into multiple lines is so that it is generally easier to read. For example, the GitHub code pane does not wrap long comment lines:

![image](https://github.com/safe-global/safe-modules/assets/4210206/243157e6-2fc5-4563-b1ba-e8ba7e5002ae)